src/revlanguage/monitors/Mntr_StochasticBranchRate.cpp

```diff
@@ -139,6 +153,7 @@ const MemberRules& Mntr_StochasticBranchRate::getParameterRules(void) const
     {
         monitor_rules.push_back( new ArgumentRule("cdbdp"          , TimeTree::getClassTypeSpec(),  "The character dependent birth-death process to monitor.",                      ArgumentRule::BY_REFERENCE, ArgumentRule::ANY, NULL ) );
         monitor_rules.push_back( new ArgumentRule("glhbdsp"        , TimeTree::getClassTypeSpec(),  "The lineage-heterogeneous birth-death process to monitor.",                    ArgumentRule::BY_REFERENCE, ArgumentRule::ANY, NULL ) );
+        monitor_rules.push_back( new ArgumentRule("bds"            , TimeTree::getClassTypeSpec(),  "The birth-death-shift process to monitor.",                                    ArgumentRule::BY_REFERENCE, ArgumentRule::ANY, NULL ) );
```

Comment:
The fact that we have separate arguments for alternative processes seems strange to me. Wouldn't it be better to have a single argument called something like `process`, overloaded so that it accepts CDBDP, GLHBDSP, and FastBDS?